client/src/css/stylesheet.scss

```diff
@@ -0,0 +1,214 @@
+@font-face {
+  font-family: "Open Sans";
+  src: url("../fonts/OpenSans-Regular.ttf") format("truetype");
+}
+
+p {
+  font-family: "Open Sans";
```

Comment:
You can probably apply this on the `html` tag instead, since it seems like you want everything to inherit this font.